bdb/bdb_int.h

```diff
@@ -1019,7 +1019,8 @@ struct bdb_state_tag {
     DB **blkseq[2];
     time_t blkseq_last_roll_time;
     DB_LSN *blkseq_last_lsn[2];
-    LISTC_T(struct seen_blkseq) blkseq_log_list[2];
+    listc_t *blkseq_log_list;
+    int pvt_blkseq_stripes;
```

Comment:
Why would you need this new variable?

Reply:
`bdb_state->pvt_blkseq_stripes` is just a copy from `bdb_state->attr->private_blkseq_stripes`. It is faster than accessing `bdb_state->attr`. It also make sure that we don't end up accessing invalid memory if  `bdb_state->attr->private_blkseq_stripes` is increased at runtime.